### ISSUE TITLE
`'culori/fn'` regression in 4.0

### ISSUE THREAD

**Issue by ai** [Evercoder/culori#223]
Seems like there is a regression in 4.0 with `'culori/fn'` import because of new `itp` color space.

Both in Node.js and browser (after Vite):

```js
import { useMode } from 'culori/fn'
```

Error:

```
Uncaught ReferenceError: Cannot access 'modeItp' before initialization
    at file:///home/ai/Dev/oklch-picker/node_modules/.pnpm/culori@4.0.0/node_modules/culori/src/index.js:274:28
```

Seems like we forgot `modeItp` in `./itp/definition.js`.

**Comment by danburzo** [comment]
Thanks for the report, @ai. Good catch! I hada meant to fix those references at code-review-time and got sidetracked, seems I forgot.

**Comment by danburzo** [comment]
Fixed in [`culori@4.0.1`](https://github.com/Evercoder/culori/releases/tag/v4.0.1), thanks for the report! It remains a small puzzle to figure out why `test/api.test.js` was not throwing this error when importing from `src/index-fn.js`…

**Comment by ai** [comment]
Thanks for quick fix. oklch.com was moved to 4.x
